index.Rmd

```diff
@@ -228,12 +230,12 @@ as(MNDWI, "SpatialPixelsDataFrame") %>%
         axis.ticks = element_blank(),
         panel.background = element_blank(),
         panel.grid.minor = element_blank()) +
-  labs(title = "Water or Water ice or Volatiles materials", 
+  labs(title = "Water or Water Ice or Volatiles Materials", 
        x = " ", 
        y = " ") +
   scale_fill_gradient(high = "#CEE50E", 
-                      low = "#087F28",
-                      name = "Water or Water ice or Volatiles materials")
+                      low = "#087F28", 
```

Comment:
The colors go from the darkest color being the smallest value and the lightest color being the largest value, which is backwards in how most humans understand colors and values. Unless there is a reason for this being opposite, I think you should switch the colors to go from lightest for smallest value to darkest for largest value. I couldn't recreate your script since it won't load past the third chunk, so I think this is the spot in your script to change your colors. I apologize if this is the wrong spot.